test/units/plugins/lookup/test_aws_secret.py

```diff
@@ -0,0 +1,88 @@
+# (c) 2019 Robert Williams
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+
+import pytest
+import datetime
+import tzlocal
```

Comment:
Instead of adding a dependency on `tzlocal`, use UTC for the timezone. Something like this should work:

```python
class UTC(datetime.tzinfo):
    ZERO = datetime.timedelta(0)

    def utcoffset(self, dt):
        return self.ZERO

    def tzname(self, dt):
        return "UTC"

    def dst(self, dt):
        return self.ZERO


utc = UTC()

value = datetime.datetime(2019, 4, 4, 11, 41, 0, 878000, tzinfo=utc)
```

See the "Example tzinfo classes" portion of the [tzinfo documentation](https://docs.python.org/2/library/datetime.html#tzinfo-objects) for the example on which this is based.

Reply:
Is there a reason not to add the dependency? The `tzlocal` library is just a wrapper around `pytz` and the [python docs](https://docs.python.org/2/library/datetime.html#tzinfo-objects) say

> The standard library has no tzinfo instances, but there exists a third-party library which brings the IANA timezone database (also known as the Olson database) to Python: pytz.
>
> pytz contains up-to-date information and its usage is recommended.

Reply:
After looking at this more closely I realized that boto3 does use local time here instead of UTC. However, boto3 is using `dateutil` instead of `tzlocal`. So this will work without any additional requirements:

```python
import dateutil.tz

value = datetime.datetime(2019, 4, 4, 11, 41, 0, 878000, tzinfo=dateutil.tz.tzlocal())
```